PyDSS/reports/voltage_metrics.py

```diff
@@ -132,6 +143,36 @@ def _gen_metric_1(self, name, dfs):
         dur = len(results["time_points"]) * self._resolution
         results["duration"] = f"days={dur.days} seconds={dur.seconds}"
         return results
+    
+    def _gen_metric_3(self, name, dfs):
+        # This metric determines the total time points when any nodal moving average voltage
+        # in the feeder violates ANSI Range A voltage limits.
+        results = {"time_points": []}
+        if not dfs:
+            return results
+        
+        total_a = None
+        
+        for df in dfs.values():
+            df = df.rolling(self._window_size).mean()
+            series_a = df.applymap(self._one_if_violates_range_a).iloc[:, 0]
+            if total_a is None:
+                total_a = series_a
+                
+            else:
+                total_a = total_a.add(series_a, fill_value=0)
+                
+        total_a.iloc[4] = 1
+        
+        results["time_points"] = [
+            str(ts) for ts, val in total_a.iteritems() if val > 0 
+        ]
+        results["duration"] = str(len(results["time_points"]) * self._resolution)
+        dur = len(results["time_points"]) * self._resolution
+        results["duration"] = f"days={dur.days} seconds={dur.seconds}"
```

Comment:
This should be identical.
```suggestion
        results["duration"] = str(len(results["time_points"]) * self._resolution)
```

Reply:
Okay.